src/app/containers/StoryPromo/index.stories.jsx

```diff
@@ -75,7 +85,10 @@ storiesOf('Containers|Story Promo/Canonical', module)
   .add('Podcast link promo', () => getCanonicalStoryPromo(podcastPromo))
   .add('Regular', () => getCanonicalStoryPromo(firstFixture))
   .add('Leading', () => getCanonicalStoryPromo(firstFixture, 'leading'))
-  .add('Top', () => getCanonicalStoryPromo(firstFixture, 'top'));
+  .add('Top', () => getCanonicalStoryPromo(firstFixture, 'top'))
+  .add('Recommendation', () =>
```

Comment:
Can recommendations promos be `LIVE`? 

At the moment we get the first item in the group:

```
const firstFixture = pathOr(
  null,
  ['content', 'groups', '0', 'items', '0'],
  fixture,
);
```
which casually has `isLive` to true.

I would suggest getting a different item from the fixture data so we pass this down to the `Regular` and `Recommendation` stories and create a new `Live promo` story passing a live item.

Reply:
Thanks @DenisHdz, yes you're right - recommendations promos can't be `LIVE`. They also do not have a `Timestamp` or `Summary` so have passed down different fixture data than the `Regular` story.

Reply:
Awesome, thank you for the updates. 

It would also be nice that the `Regular`, `Leading` and `Top` stories use a different item from the Pidgin feed that is not live, so we could add another story showing a `Live promo` passing explicitly a live item.

However, since this is out of the scope of this PR, I'm happy to approve it already :)